test/playback.test.js

```diff
@@ -59,6 +59,27 @@ QUnit.test('Advanced Bip Bop', function(assert) {
   });
 });
 
+QUnit.skip('Fmp4', function(assert) {
+  let done = assert.async();
+
+  assert.expect(2);
+  let player = this.player;
+
+  player.autoplay(true);
+
+  playFor(player, 6, function() {
+    assert.ok(true, 'played for at least six seconds order over to hit the change in codec');
```

Comment:
I think "order over" can be removed. And it should just be a change in container format, the codec should remain the same.